gateway-ha/src/main/java/io/trino/gateway/ha/module/NotifierModule.java

```diff
@@ -1,40 +0,0 @@
-/*
```

Comment:
Removing this class will eventually have to be dealt with so we can bring this functionality back for the pending PR about email notifications.. assuming we want that

Reply:
I saw #244 is using ClusterStateListenerModule, not NotifierModule.

Reply:
Great.. no issue then.